### ISSUE TITLE
This addon seems to have broken with Webthings 1.0 upgrade

### ISSUE THREAD

**Issue by lacojim** [flatsiedatsie/highlights#4]
All the devices which use this are showing as Disconnected, after the Webthings 1.0 update. Not sure how to troubleshoot it further.

**Comment by flatsiedatsie** [comment]
Yes, thanks. I have to update all addons to use the new path. On it.

**Comment by madb1lly** [comment]
Hi @flatsiedatsie,

After update to webthings 1.0 and migration to WebThings etc from Mozilla-iot the highlights adapter was empty.  I thought the token might be invalid so i replaced it which didn't work, so I just readded all to the highlights and they came back. 

Until today when they had all disappeared again 😆 i've readded them again, but I presume they'll disappear again until you fix the adapter,  right? 

Cheers 🙂

**Comment by flatsiedatsie** [comment]
Yes. It should be fixed now.
